bin/diagnostic.js

```diff
@@ -1 +1,14 @@
 'use strict'
+const ask = require('../lib/ask.js')
+
+let answer = 'not empty'
+
+while (answer !== '') {
```

Comment:
Make sure you actually enter a number for the user to guess in addition to checking if their answer was between 1-10.